test/test_charm.py

```diff
@@ -25,53 +23,18 @@
 
 import ops
 import ops.charm
-from ops.model import ModelError, _ModelBackend
-from ops.storage import SQLiteStorage
+from ops.model import ModelError
 
-from .test_helpers import fake_script, fake_script_calls
+from .test_helpers import FakeScript, create_framework
 
 
-class TestCharm(unittest.TestCase):
+@pytest.fixture
+def fake_script_fixture(monkeypatch: pytest.MonkeyPatch, tmp_path: pathlib.Path):
```

Comment:
If I move this to test_helpers, lint will show an error about unused import in the test cases, so I'm keeping it here.

Reply:
Hmm, is there a better way to do handle this? This will be repeated in about 5 different test files, so it'd be nice if we can avoid it. Is there a standard way to import pytest fixtures? Or should we just silence the linter error?

Reply:
Also, do you think we should just call this `fake_script`? That seems nicer than the lengthy `fake_script_fixture` in the tests. It seems like fixtures don't normally have `fixture` in the name.

Reply:
I have figured out a way to share fixtures across multiple files. I will change the name to `fake_script`, too, shorter name is better and the `_fixture` suffix seems unnecessary according to standard pytest fixtures.